assets/javascripts/ng-app/services/car_select.js

```diff
@@ -3,6 +3,8 @@ angular.module('myApp')
   var that = this;
   var makesUrl = '/api/v1/makes';
 
+  this.waitIcon = true;
```

Comment:
You'll notice in the current code that we store certain values in the service. This is necessary if you want to store values after someone clicks away. In this case, the loading gif's status doesn't need to be stored so don't need to set it at the service level.